azure-pipelines.yml

```diff
@@ -98,6 +100,7 @@ jobs:
   - script: |
       pushd src/mainboard/emulation/qemu-riscv
```

Comment:
instead of pushd/popd could you put this in ()
(cd bla && make && make blah blah)
?